### DISCUSSION TITLE
Student repositories 2021

### DISCUSSION THREAD

**Discussion by mambax** [nds-swe/spring-starter#2] [Show and tell]
Listing all starter repositories to show and tell.

**Comment by mambax** [comment]
https://github.com/nds-swe/spring-starter

**Comment by KlausBuderer** [comment]
https://github.com/KlausBuderer/SpringStarter.git

**Comment by devdbl** [comment]
https://github.com/devdbl/spring-starter

**Comment by bruno-thurnherr** [comment]
https://github.com/bruno-thurnherr/starter

**Comment by micheladirian** [comment]
https://github.com/micheladirian/Spring-Starter/tree/master/spring-starter-sample-solution-1.0.0

**Reply by micheladirian** [reply]
Ist leider secure probiere dies noch zu ändern

**Comment by Peschehalter** [comment]
https://github.com/Peschehalter/Spring-Starter
